### PR TITLE
Rename OpenShift GitOps default Argo CD instance name in KAM

### PR DESCRIPTION
**Pull request by ishitasequeira**
**What type of PR is this?**
 /kind code-refactoring

**What does this PR do / why we need it**:
The default Argo CD instance name would be changed to **openshift-gitops**  when the https://github.com/redhat-developer/gitops-operator/pull/70 is merged. The associated change has been made in KAM

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:
https://issues.redhat.com/projects/GITOPS/issues/GITOPS-776 

Fixes #?

**How to test changes / Special notes to the reviewer**:
Perform the Day 1 operations. All the app should be synced in Argo CD.

### CODE REVIEW

**Comment by wtam2018** [comment]
Please grep for "argocd-cluster".  It looks like there are still a couple of reference of the old  name in README.md.   
Have to test it with bootstrapping and Day1?

File: scripts/setup-operators.sh [hunk 1]
```diff
@@ -104,4 +104,4 @@ done
 echo "Completed OpenShift GitOps operator installation"
 
 echo "Provide cluster-admin access to argocd-application-controller service account"
-oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:openshift-gitops:argocd-cluster-argocd-application-controller
+oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:openshift-gitops:openshift-gitops-argocd-application-controller
```
Comment (chetan-rns) [inline scripts/setup-operators.sh:107]: We no longer have to make Argo CD cluster-admin. We can fix it in a different PR

**Review by chetan-rns — APPROVED** [review]
Thanks for PR @ishitasequeira 
/lgtm

File: scripts/setup-operators.sh (hunk 1 above)
Comment (amitkrout) [reply scripts/setup-operators.sh:107]: > We no longer have to make Argo CD cluster-admin.

This step has been deprecated in day 1 doc too. @ishitasequeira Please remove it

> We can fix it in a different PR

@chetan-rns Do we have jira or github issue ?, if not please create one

Comment (chetan-rns) [reply scripts/setup-operators.sh:107]: Will create a new issue @amitkrout

**Pull request merged**